src/github.com/getlantern/fronted/masquerade.go

```diff
@@ -137,10 +137,13 @@ func (vms *verifiedMasqueradeSet) doVerify(masquerade *Masquerade) bool {
 	go func() {
 		start := time.Now()
 		httpClient := vms.dialer.HttpClientUsing(masquerade)
-		req, _ := http.NewRequest("HEAD", "http://www.google.com/humans.txt", nil)
+		req, err := http.NewRequest("HEAD", "http://www.google.com/humans.txt", nil)
+		if err != nil {
+			errCh <- fmt.Errorf("http.NewRequest Error: %s", err)
+		}
 		resp, err := httpClient.Do(req)
 		if err != nil {
-			errCh <- fmt.Errorf("HTTP ERROR FOR MASQUERADE %v: %v", masquerade.Domain, err)
+			errCh <- fmt.Errorf("HTTP ERROR FOR MASQUERADE %v: %v, tlsInfo: %s", masquerade.Domain, err, vms.dialer.tlsInfo(masquerade))
```

Comment:
> To avoid adding unrelated info to Loggly, instead of print tlsInfo for any errors, can we only print it if the error isx509.UnknownAuthorityError

Good idea!